### PR TITLE
[rfr]Support Extra routes extension in router api

### PR DESCRIPTION
**Pull request by timbyr**
http://developer.openstack.org/api-ref-networking-v2-ext.html#extraroute-ext

### CODE REVIEW

File: openstack/networking/v2/extensions/layer3/routers/requests.go [hunk 1]
```diff
@@ -97,6 +97,7 @@ type UpdateOpts struct {
 	Name         string
 	AdminStateUp *bool
 	GatewayInfo  *GatewayInfo
+	Routes       *[]Route
```
Comment (jrperritt) [inline openstack/networking/v2/extensions/layer3/routers/requests.go]: Slices are pointers underneath, so we can just have `[]Route` here.

**Comment by jrperritt** [comment]
@timbyr Just one small change, then we can merge.

**Comment by hingstarne** [comment]
Hi, 
are there any news on that. We could really have a big advantage of having these feature.
Cheers

**Comment by Fodoj** [comment]
+1 - would be a-mazing to see this PR merged

**Comment by jrperritt** [comment]
Sorry for the delay; this looks good. +2

**Pull request merged**